sklearn/kernel_ridge/kernel_ridge.py

```diff
@@ -69,7 +51,7 @@ class KernelRidge(BaseEstimator, RegressorMixin):
 
     Attributes
     ----------
-    dual_coef_ : array, shape = [n_samples] or [n_samples, n_targets]
+    F : array, shape = [n_samples] or [n_samples, n_targets]
```

Comment:
what is this? I can't see where you set F. You shouldn't be dropping `dual_coef_` either.

Reply:
On the other hand, we don't really need a docstring in the base class

Reply:
Sorry about F/dual_coef_, I don't know why that was there. Now it's fixed.

I added the docstring because:
- When I was running the tests, some of them failed because this object had `def __init__` but no docstring, so I had to added.
- I think it is useful and it does not disturb.

But if you are really concerned about this, I can remove it. But I think there is one test that would complain.